backend/src/database/seeds/08_picker_options.js

```diff
@@ -38,6 +41,34 @@ exports.seed = function (knex) {
       definition: { tableLookup: "amendment_type" },
       associated_form: tables.contract_amendment,
     },
+    {
+      name: "status",
```

Comment:
i tried to put these in alpha order, just to make it easier to find.

Reply:
Ah I didn't notice that I'll reorder them to keep them alphabetical then.